nilearn/datasets/func.py

```diff
@@ -870,7 +878,7 @@ def fetch_localizer_contrasts(contrasts, n_subjects=None, get_tmaps=False,
                   "X concerns S, "
                   "X label XL, X identifier XI, "
                   "X format XF, X description XD, "
-                  'S identifier <= "%s", ' % (subject_id_max, ) +
+                  'S identifier IN %r, ' % (tuple(subject_ids), ) +
```

Comment:
@AlexandreAbraham : I'd be interested to know if you think that this change is the right thing, or whether it would have been better to leave it as before.